layouts/community/ortho_4x12/mindsound/config.h

```diff
@@ -0,0 +1,31 @@
+#ifndef CONFIG_USER_H
```

Comment:
Could you replace the ifndef and the define here with just `#pragma once`, and remove the endif at the end?

Reply:
aye aye